src/utils/compareDate.js

```diff
@@ -0,0 +1,9 @@
+import { MILLI_SECONDS_IN_A_DAY } from "src/common/constants";
+
+export function getDaysToCompleteTask(startDate, endDate) {
+  const millisecondsInADay = MILLI_SECONDS_IN_A_DAY;
+  const diffInMilliseconds = Math.abs(endDate - startDate);
+  const diffInDays = Math.round(diffInMilliseconds / millisecondsInADay) + 1;
+
```

Comment:
Could you please modify the code to use Math.floor() instead of Math.round() for calculating the number of days it takes to complete the task, as we want to get the whole number of days rather than the closest approximation?